Tasks/AzureSpringCloudV0/operations/AzureResourceFilterUtility.ts

```diff
@@ -0,0 +1,23 @@
+import tl = require('azure-pipelines-task-lib/task');
+import { AzureEndpoint } from 'azure-pipelines-tasks-azure-arm-rest-v2/azureModels';
+import { Resources } from 'azure-pipelines-tasks-azure-arm-rest-v2/azure-arm-resource';
+
+export class AzureResourceFilterUtility {
+    public static async getAzureSpringCloudResourceId(endpoint: AzureEndpoint, resourceName: string): Promise<string> {
+        tl.debug('Looking up Azure Spring Cloud Instance ' + resourceName);
+        var azureResources: Resources = new Resources(endpoint);
+        var filteredResources: Array<any> = await azureResources.getResources('Microsoft.AppPlatform/Spring', resourceName);
```

Comment:
As a good practise, use `const` for objects that are not expected to be modified, `let` for objects that may get modified. Prefer `let` over `var`.

Reply:
Fixed in a places where it was straightforward.